prestoadmin/server.py

```diff
@@ -772,69 +773,69 @@ def print_status_header(external_ip, server_status, host):
 
 @parallel
 def collect_node_information():
-    client = PrestoClient(get_coordinator_role()[0], env.user)
-    with settings(hide('warnings')):
-        error_message = check_presto_version()
-    if error_message:
-        external_ip = 'Unknown'
-        is_running = False
-    else:
-        with settings(hide('warnings', 'aborts', 'stdout')):
-            try:
-                external_ip = get_ext_ip_of_node(client)
-            except:
-                external_ip = 'Unknown'
-            try:
-                is_running = service('status')
-            except:
-                is_running = False
-    return external_ip, is_running, error_message
-
-
-def get_status_from_coordinator():
-    client = PrestoClient(get_coordinator_role()[0], env.user)
-    try:
-        coordinator_status = run_sql(client, SYSTEM_RUNTIME_NODES)
-        catalog_status = get_catalog_info_from(client)
-    except BaseException as e:
-        # Just log errors that come from a missing port or anything else; if
-        # we can't connect to the coordinator, we just want to print out a
-        # minimal status anyway.
-        _LOGGER.warn(e.message)
-        coordinator_status = []
-        catalog_status = []
-
-    with settings(hide('running')):
-        node_information = execute(collect_node_information,
-                                   hosts=get_host_list())
-
-    for host in get_host_list():
-        if isinstance(node_information[host], Exception):
+    with closing(PrestoClient(get_coordinator_role()[0], env.user)) as client:
```

Comment:
Actually, I think this is just a super misleading diff caused by the whitespace change. Please look extra-close :-(